html/introduction-to-html/getting-started/index.html

```diff
@@ -4,8 +4,97 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width">
     <title>My test page</title>
+    <style>
+      form {
```

Comment:
Add the styles to an external styles file then link it to this html file

Reply:
Please follow the instructions to add a Pull Request Template on your Week Two card tasks :) @sharon-kathambi